snmp_types.yaml

```diff
@@ -0,0 +1,87 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.2m4/types.yaml
+  - http://www.getcloudify.org/spec/diamond-plugin/1.2m4/plugin.yaml
+  - http://www.getcloudify.org/spec/openstack-plugin/1.2m4/plugin.yaml
+
+
+node_types:
+  SNMPSecurityGroup:
+    derived_from: cloudify.openstack.nodes.SecurityGroup
+    properties:
+      security_group:
+        default:
+          name: snmp_security_group
+          description: Allows UDP communication for SNMP
+      rules:
+        default:
+          - remote_ip_prefix: 0.0.0.0/0
+            port: 161
+            protocol: udp
+
+  SNMPProxy:
+    derived_from: cloudify.nodes.Root
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
```

Comment:
If create takes no inputs perhaps it would be best if the shorter form is used
`create: scripts/install_requirements.sh`